### ISSUE TITLE
FutureWarning message of v1.6.3.

### ISSUE THREAD

**Issue by zhaoc1** [MrOlm/inStrain#115]
Hi Matt,

I tried to run inStrain v1.6.3. installed from Conda with the command `inStrain profile`, and got the following warning message:

```
/home/ubuntu/miniconda3/envs/instrain/lib/python3.8/site-packages/inStrain/profile/snv_utilities.py:284: FutureWarning: In a future version, `df.iloc[:, i] = newvals` will attempt to set the values inplace instead of always setting a new array. To retain the old behavior, use either `df[df.columns[i]] = newvals` or, if columns are non-unique, `df.isetitem(i, newvals)`
  SNPTable.loc[:,'cryptic'] = SNPTable['cryptic'].fillna(False)
```

The Warning message seems to be related with the number of split, which means it showed up way too many time. 

Is there a quick fix for this warning?

Thanks,
Chunyu

**Comment by MrOlm** [comment]
Hi Chunyu,

Stupid warnings like this pop up all the time when updating dependencies like numpy and scipy. Quite annoying. I'll fix on the next update.

Thanks for letting me know,
Matt

**Comment by MrOlm** [comment]
Fixed in v1.7.1
